engine/database/output.go

```diff
@@ -0,0 +1,19 @@
+package database
+
+// Output represents an output of a transaction.
+// Fixme: This is not integrated with out db engine yet.
+type Output struct {
+	TxID       string  `gorm:"primaryKey"`
+	Vout       uint32  `gorm:"primaryKey"`
```

Comment:
Maybe we could make this an outpoint (hmmm.. but from the other side, maybe it's better to have only plain columns at this package with relationships and convert them in other packages)

Reply:
I would stick to "only plain columns".